genienlp/arguments.py

```diff
@@ -472,26 +480,29 @@ def parse_argv(parser):
         'ngrams: lookup all ngrams in the text and see if there is a match',
     )
 
-    parser.add_argument('--verbose', action='store_true', help='Print detected types for each token')
     parser.add_argument(
         '--almond_domains', nargs='+', default=[], help='Domains used for almond dataset; e.g. music, books, ...'
     )
     parser.add_argument(
         '--ned_features',
         nargs='+',
         type=str,
-        default=[],
-        help='Features that will be extracted for each entity. Order is important',
+        default=['type_id', 'type_prob', 'qid'],
+        help='Features that will be extracted for each entity: "type" and "qid" are supported. Order is important',
```

Comment:
We're changing the default here: what's the impact for current training configurations with and without Bootleg?

Reply:
For experiments that don't use ned (`do_ned` is False) there's no impact. 
For NED experiments, all features are processed by default. Now, If you want to use the features you would call `add_types_to_text` and/ or `add_qid_to_text`. I'm working on second cleanup for the code that we'll drop the `add_*_to_args` and add an argument to tell if we are adding features as text to the input or embedding ids. And then whatever features is passed as `ned_features` will be used automatically.